mw-webapp/src/component/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import {ReactElement, useState} from "react";
+import {
+  Content as DropdownContent,
+  Root as DropdownRoot,
+} from "@radix-ui/react-dialog";
+import {
+  DropdownMenuItem,
+  DropdownMenuItemType,
+} from "src/component/dropdown/dropdownMenuItem/DropdownMenuItem";
+import styles from "src/component/dropdown/Dropdown.module.scss";
+
+/**
+ * Dropdown props
+ */
+export interface DropdownProps {
+
+  /**
+   * The element that triggers the Dropdown.
+   */
+  trigger: ReactElement<HTMLElement>;
+
+  /**
+   * DropdownMenuItems list
+   */
+  dropdownMenuItems: DropdownMenuItemType[];
+}
+
+/**
+ * Dropdown conponent
+ */
+export const Dropdown = (props: DropdownProps) => {
+  const [isOpenMenu, setIsOpenMenu] = useState(false);
+
+  const renderDropdownMenuItems = props.dropdownMenuItems.map((item) => {
+    const {id, value, onClick, isVisible = true} = item;
```

Comment:
We are trying to avoid using destructuring, let's try to get rid of it. With destructure you need to write less, but think a little bit more (because of lost context, for example: it will be harder to differentiate state and props variables without context).

Reply:
fixed